master/internal/configpolicy/postgres_task_config_policy.go

```diff
@@ -113,3 +114,55 @@ func DeleteConfigPolicies(ctx context.Context,
 	}
 	return nil
 }
+
+// GetEnforcedConfig gets the fields of the global invariant config or constraint if specified, and
+// the workspace invariant config or constraint otherwise. If neither is specified, returns nil.
+func GetEnforcedConfig[T any](ctx context.Context, wkspID *int, policyType, field, workloadType string) (*T,
+	error,
+) {
+	if policyType != "invariant_config" && policyType != "constraints" {
+		return nil, fmt.Errorf("invalid policy type :%s", policyType)
+	}
+
```

Comment:
Please also validate `workloadType`; I think all our other postgres functions do. There's no need to add a test case for it.

Reply:
it seems like we actually don't validate `workloadType` in any of the postgres functions! At first this seemed odd, but then I remembered we made `workload_type` an [enum](https://github.com/determined-ai/determined/blob/main/master/static/migrations/20240829000038_add-task-config-policy-table.tx.up.sql#L1)!
I can still perform the validation if you'd like, but this function would be unique in that regard

Reply:
In that case, don't add it. I was mistaken!